composer.json

```diff
@@ -21,6 +21,7 @@
 	},
 	"require": {
 		"php": ">=7.1",
+		"altis/browser-security": "~1.0",
```

Comment:
Need to do a release and then good to go. However less keen on this style of version constraint, I've been using `^1.0.0` even for our own stuff

Reply:
Yep, hence Draft. I'll change that constraint, wasn't sure which we should go for.